generate_PSC.py

```diff
@@ -39,20 +40,29 @@
     obstime = obstime + gap
     metadata['TIME{:04d}'.format(t)] = obstime.isoformat()
 hdu.header.extend(metadata)
+print(metadata)
 
-hdu.writeto('PSC_0001.fits', clobber=True)
+hdu.writeto('PSC_0002.fits', clobber=True)
 
 ## Generate Fake Lightcurve
-with open('PSC_0001.dat', 'w') as FO:
-    FO.write('# {:24s} {:6s} {:6s} {:6s} {:6s} {:6s} {:6s}\n'.format(
-             'Time', 'R', 'G', 'B', 'sig_R', 'sig_G', 'sig_B'))
+with open('PSC_0002.json', 'w') as FO:
+    data = []
     for t in range(nt):
         time = hdu.header['TIME{:04d}'.format(t)]
         sig_r = 0.010
         sig_g = 0.006
         sig_b = 0.017
-        r = np.random.normal(1,sig_r)
-        g = np.random.normal(1,sig_g)
-        b = np.random.normal(1,sig_b)
-        FO.write('{:26s} {:6.3f} {:6.3f} {:6.3f} {:6.3f} {:6.3f} {:6.3f}\n'.format(
-                 time, r, g, b, sig_r, sig_g, sig_b))
+        r = np.random.normal(1, sig_r)
+        g = np.random.normal(1, sig_g)
+        b = np.random.normal(1, sig_b)
+        entry = {
```

Comment:
We should figure out how to format and add metadata (clone of the fits file header?) to this JSON entry.